### PR TITLE
BLOC-42 Refactor Notification component

### PR DESCRIPTION
**Pull request by almondy22**

### CODE REVIEW

File: app/components/Notifications/Notification.jsx [hunk 1]
```diff
@@ -24,250 +16,106 @@ import {
...
   NOTIFICATIONS_TYPES,
-  ROW_HEIGHT,
 } from './constants';
-
-import Span from '../Span';
+import styles from './Notifications.styled';
 
 const single = isSingleCommunityWebsite();
-const styles = singleCommunityStyles();
+const singleStyles = singleCommunityStyles();
```
Comment (AweAwe95) [inline app/components/Notifications/Notification.jsx:26]: communityStyles

File: app/components/Notifications/Notification.jsx [hunk 2]
```diff
@@ -24,250 +16,106 @@ import {
...
         .split(' ')
         .map((x, i) => (i === 0 ? trimRightZeros(x) : x))
         .join(' '),
     };
-  }, [data]);
-
-  const isCommunityMod = Boolean(single) && Object.keys(styles).length > 0;
+  }, [data.quantity, isTippedType]);
 
+  const isCommunityMod =
```
Comment (AweAwe95) [inline app/components/Notifications/Notification.jsx:86]: isCommunityMode

File: app/components/Notifications/Notifications.styled.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  containerStyles: {
+    position: 'absolute',
+    padding: '0 18px',
+    fontSize: '16px',
+    width: '100%',
+    display: 'flex',
+    flexDirection: 'column',
+    borderBottom: `1px solid ${BORDER_SECONDARY_LIGHT}`,
+    alignItems: 'start',
+    justifyContent: 'center',
```
Comment (AweAwe95) [inline app/components/Notifications/Notifications.styled.ts:13]: you can use className="pa fz16 df fdc aic jcc" in Notification.jsx => same for all styled files

File: app/containers/Header/NotificationsDropdown/Menu/Notification.jsx [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+    }
+    return {
+      quantity: data.quantity
+        .split(' ')
+        .map((x, i) => (i === 0 ? trimRightZeros(x) : x))
+        .join(' '),
+    };
+  }, [data.quantity, isTippedType]);
+
+  const isCommunityMod =
```
Comment (AweAwe95) [inline app/containers/Header/NotificationsDropdown/Menu/Notification.jsx:85]: Mode

File: app/containers/Header/NotificationsDropdown/Menu/Notification.jsx [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+    <div
+      css={{
+        ...styles.root,
+        ...(!read ? styles.unreadStyles : {}),
+        ...(isLast && { border: 'none' }),
+        height: `${height}px`,
+        top: `${top}px`,
+      }}
+    >
+      <IconMd
```
Comment (AweAwe95) [inline app/containers/Header/NotificationsDropdown/Menu/Notification.jsx:103]: u can use Icon Components

**Review by YuriyLu — APPROVED** [review]

File: app/components/Notifications/Notification.jsx (hunk 2 above)
Comment (almondy22) [reply app/components/Notifications/Notification.jsx]: Fixed

File: app/containers/Header/NotificationsDropdown/Menu/Notification.jsx (hunk 5 above)
Comment (almondy22) [reply app/containers/Header/NotificationsDropdown/Menu/Notification.jsx]: Done

**Review by AweAwe95 — APPROVED** [review]

**Pull request merged**